frontend/src/components/common/Terminal.tsx

```diff
@@ -210,6 +223,11 @@ export default function Terminal(props: TerminalProps) {
       } else {
         xterm.write(t('Failed to connect…') + '\r\n');
       }
+
+      xterm.write('\r\n' + t('Press the enter key to reconnect.') + '\r\n');
```

Comment:
Is there a reason for doing this in the terminal, and not with... say a mui dialog/button?

Reply:
I did have a simple "Reconnect" button, but that's new UI we're adding and I'd like to do that in addition to allowing the user to set up the shell binary to run. So this is an unintrusive change IMO (we were already writing into the terminal when errors were found).